cloup/_option_groups.py

```diff
@@ -185,19 +185,25 @@ def get_ungrouped_options(self, ctx: click.Context) -> Sequence[click.Option]:
 
     def get_argument_help_record(
         self, arg: click.Argument, ctx: click.Context
-    ) -> Tuple[str, str]:
+    ) -> Optional[Tuple[str, str]]:
         if isinstance(arg, cloup.Argument):
-            return arg.help_record(ctx)
+            return arg.get_help_record(ctx)
         return arg.make_metavar(), ""
 
     def get_arguments_help_section(self, ctx: click.Context) -> Optional[HelpSection]:
         if not any(getattr(arg, "help", None) for arg in self.arguments):
             return None
+        help_records = list(
```

Comment:
I understand you're just being consistent with click and the semantic of `get_help_record` but `get_argument_help_record` never returns `None` and can't imagine a reason for it to return `None`. Click uses this semantic to filter out arguments and to avoid showing options with `hidden = True`. A positional argument, on the other end, should never be hidden. 

Let me know if I'm missing something!

Reply:
@janluke Ah, I see how you are doing it now. Fair enough. I think the problem was caused by Typer supporting hidden arguments. Which is pretty unnatural, but it still supports them! I'll have a look into just supporting that on the Typer end.

Reply:
Great!